downstream/assemblies/eda/assembly-eda-credentials.adoc

```diff
@@ -1,10 +1,15 @@
 [id="eda-credentials"]
 
-= Setting up credentials for {EDAcontroller}
+= Credentials
+
+You can use credentials and credential types to define and store secrets that can be used for authentication purposes with resources, such as decision environments, rulebook activations and projects for {EDAcontroller}, and inventories and projects for {ControllerName}.
```

Comment:
@jself-sudoku We don't have inventories in eda, I think we can safely remove "and inventories".
Also in the first line should we switch the credential types and credentials to something like this.
```
You can use credential types and credentials to define and store secrets
```
credential type => define
credentials => store secrets

Reply:
Thanks, @mkanoor . This was an oversight. I started out combining both credentials and credential types in the same chapter, but decided to break them out into two chapters instead. I will correct this statement.